modules/security-cors/src/main/java/com/devonfw/module/security/cors/common/impl/CorsConfigProperties.java

```diff
@@ -0,0 +1,113 @@
+package com.devonfw.module.security.cors.common.impl;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * {@link ConfigurationProperties} to configure Cross-Origin Resource Sharing (CORS).
+ *
+ * @since 2020.12.002
+ */
+@Configuration
+@ConfigurationProperties(prefix = "security.cors")
+public class CorsConfigProperties {
```

Comment:
Lets change this entire class to this:
```
@Configuration
@ConfigurationProperties(prefix = "security.cors")
public class CorsConfigProperties extends CorsConfiguration {
}
```
This will give us all config options from springs `CorsConfiguration` and even future ones added in spring without changing our code.